scripts/e2e/e2e-datadog-logs/test.yaml

```
@@ -23,5 +25,5 @@ paths:
 - "src/internal_events/datadog_*"
 - "src/sinks/datadog/logs/**"
 - "src/sinks/util/**"
-- "scripts/integration/datadog-e2e/logs/**"
+- "scripts/e2e/e2e-datadog-logs/**"
```

Tiny nit - `e2e-datadog-logs` can now be `datadog-logs` given its directory.
```suggestion
- "scripts/e2e/datadog-logs/**"
```

---

Oh I meant to add a note about that . In the vdev parsing, without taking on the changes in the new GH issue to create the `e2e` subcommand, we dont have a way to distinguish between whether someone meant the `datadog-logs` integration test, or the `datadog-logs` e2e test, so I had to add that prefix as an intermediary. I can add a comment.

---

addressed in https://github.com/vectordotdev/vector/pull/18538/commits/cde2a2d1065a4cd6cc666db6c70ceea8d4b5aed2